app/admin/monthly_report.rb

```diff
@@ -0,0 +1,18 @@
+ActiveAdmin.register MonthlyReport do
+
+# See permitted parameters documentation:
+# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+#
+# permit_params :list, :of, :attributes, :on, :model
+#
+# or
+#
+# permit_params do
+#   permitted = [:permitted, :attributes]
+#   permitted << :other if resource.something?
+#   permitted
+# end
+
+  PERMIT_COLUMUNS  = %w(user_id project_summary used_technology responsible_business business_content looking_back next_month_goals month year)
```

Comment:
__[rubocop]__
* Unnecessary spacing detected.
* Line is too long. [143/128] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)